### DISCUSSION TITLE
running LAPS-WebUI V1.5.6 x64

### DISCUSSION THREAD

**Discussion by nyxfpv** [Seji64/LAPS-WebUI#19] [General]
Windows 10 
.net core 7.0
I configured the appsetting, and when I rin the Laps-WebUI.exe, all I get is a DOS window with a blinking cursor with no errors 
I tried to access the site page and it times out.

**Comment by Seji64** [comment]
Hello,

I agree with you, it's not optimal that there is no logging at startup (I actually only use it myself with docker 😇). However, it works without problems in my test - laps web ui listens at `http://127.0.0.1:5000/` by default

**Comment by nyxfpv** [comment]
I was able to get it running...    thanks

**Comment by nyxfpv** [comment]
Question is there a way to have it start automatically when Windows starts?

**Reply by Seji64** [reply]
You could run it as Service using nssm
